nexus-sdk-js/src/Project/index.ts

```diff
@@ -97,7 +97,7 @@ export default class Project {
 
   async getResource(id: string): Promise<Resource> {
     try {
-      return await Resource.getSelf(id, this.orgLabel, this.label);
+      return await Resource.get(this.orgLabel, this.label, null, id);
```

Comment:
Why not simply call ` Resource.get(this.orgLabel, this.label, '_', id);` ?

Reply:
+1, if using `'_'` is enough to get the resource, why add the `null` special case below? Only this one line could change